assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml

```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates RDS Cluster
+Resources:
+  RDSCluster: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      MasterUserPassword: 
+        Ref: password
+      MasterUsername: 
+        Ref: username
+      StorageEncrypted: true  
+    Type: "AWS::RDS::DBCluster"
+  RDSDBClusterParameterGroup: 
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```

Comment:
Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`